Purchases/Networking/Operations/NetworkOperation.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  NetworkOperation.swift
+//
+//  Created by Joshua Liebowitz on 11/18/21.
+
+import Foundation
+
+class CacheableNetworkOperation: NetworkOperation, Cachable {
+
+    var key: String { "\(Self.description()) \(individualizedCacheKeyPart)" }
```

Comment:
instead of relying on the API url path as part of the cache key, we use the type of the Operation now.

Reply:
What do you think about using `type(of: self)` instead?

Reply:
Yup, totally think that's better.